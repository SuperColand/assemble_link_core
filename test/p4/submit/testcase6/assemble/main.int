.data
0

.text
addiu $a0 $0 0
addiu $a0 $0 37
jal prime
syscall
