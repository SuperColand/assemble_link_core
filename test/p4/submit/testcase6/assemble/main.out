.data
0

.text
24040000
24040025
0c000000
0000000c

.symbol
0	main

.relocation
8	prime
